ats-cert-parser/src/lib.rs

```diff
@@ -0,0 +1,311 @@
+use serde::{Deserialize, Serialize};
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Creator {
+    fullname: String,
+    email: String,
+    roles: Vec<String>,
+    ipi: String,
+    isni: String,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct CreatorInput {
+    pub fullname: String,
+    pub email: String,
+    pub roles: Vec<String>,
+    pub ipi: String,
+    pub isni: String,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct AtsCertificateInput {
```

Comment:
What the reason to use a duplicated definition of Creator and AtsCertificate here ?

As the definition look literally the same, better to use directly the types Creator and AtsCertificate so ?

Reply:
I will only import Creator as `AtsCertificateInput` from the parser and `CertificateData` from the generator serves different purposes. I'll update `CertificateData` to accept datas created by the ZKP circuits sush as all the different `hash`, the `secret` and the `id_allfeat`. I should better name this types tho. Does it sound good to you ?

Reply:
Good, as my point was, just avoid defining two structs which have the exact same definition and prefer type aliases to rename a type to adapt to the logic you wanna use for.